### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by bethanyios**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -2,34 +2,61 @@
...
+
+- [Finding a Tech Partner](extra-resources/finding-a-tech-partner.md)
+- [Glossary](extra-resources/glossary.md)
+
+
+
+
+
 
 ## Core Modules
```
Comment (Albadylic) [inline SUMMARY.md:53]: Could we remove reference to core modules and pop all exercises in the list?

File: list-of-exercises/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+description: >-
+  In this section of the GitBook you will find a list of exercises that can be used in design sprint workshops.
+---
+
+# The List of Exercises
+
+Each of these exercises are adaptable and can be reorganised depending on the needs of the group you are facilitating. Tailoring the workshop by cherry picking specific exercises and arranging them in a certain order is a really powerful way of helping your user group process their problem areas and solutions. Your workshops become bespoke but still follow the design sprint structure.
```
Comment (Albadylic) [inline list-of-exercises/README.md:8]: Could we remove reference to facilitating? Iirc, this page may be read by charities too?

**Review by Albadylic — CHANGES_REQUESTED** [review]
Thanks for this! Much cleaner structure. I have a couple of wee suggestions and generally, I think this looks good and should work for GB.

File: SUMMARY.md [hunk 3]
```diff
@@ -2,34 +2,61 @@
...
+- [Exercises](list-of-exercises/README.md)
+- [The Five Whys](list-of-exercises/the-five-whys.md)
+- [The Bigger Picture](list-of-exercises/the-bigger-picture.md)
+- [Problem Statement](list-of-exercises/problem-statement.md)
+- [How Might We](list-of-exercises/how-might-we.md)
+- [Mapping](list-of-exercises/mapping.md)
+- [Assumptions Table](list-of-exercises/assumptions-table.md)
+- [Assumptions to Questions](list-of-exercises/assumptions-to-questions.md)
+- [Common Areas](list-of-exercises/common-areas.md)
+- [User Research Scripting](core-modules/discovery-workshop/user-research-scripting.md)
```
[the remark this answers is not in the document]
Comment (bethanyios) [reply SUMMARY.md:31]: Good spot that was my mistake

File: SUMMARY.md (hunk 1 above)
Comment (bethanyios) [reply SUMMARY.md:53]: All exercises are in the list but I didn't want to remove anything yet - will remove references in the summary though

File: list-of-exercises/README.md (hunk 2 above)
Comment (bethanyios) [reply list-of-exercises/README.md:8]: Yes!

Comment (bethanyios) [reply list-of-exercises/README.md:8]: This isn't complete as I was just focusing on the structure so we can fix content later.

File: SUMMARY.md (hunk 1 above)
Comment (Albadylic) [reply SUMMARY.md]: Ah, what I meant was moving User Personas etc. up into the main list - i.e. removing the headline "Core Modules" and "Discovery Workshop" but keeping the exercise itself.

**Review by Albadylic — APPROVED** [review]

**Review by jc2820 — COMMENTED** [review]
Imo the 'Case Studies' and 'List of Exercises' titles should lead directly to their respective README.md if clicked, and the rest of the files should be dropdown children of that link.

Also, do we need 2 markdown files for each case study?

**Pull request merged**